lewis/scripts/control.py

```diff
@@ -48,7 +48,7 @@ def show_api(remote, object_name):
             raw_value = str(getattr(obj, prop))
             value_lines = raw_value.split('\n')
 
-            current_value = value_lines[0][40:] + (
+            current_value = value_lines[0][:40] + (
```

Comment:
Good catch, really weird that we did not see it earlier. But it's not enough to just change this, now all the values are gone if the string is below 40 characters. It took me a while to see that and I still don't quite understand what's going on,